web/src/lib/utils/helpers.ts

```diff
@@ -45,6 +45,8 @@ export const useMountEffectWhenDefined = (fun: () => void, thingToBeDefined: unk
 
 export const useScrollToPathAnchor = (): void => {
   useEffect(() => {
+    if (window.location.pathname === "/mgmt/cms") return;
```

Comment:
Scroll hook was trying to use the admin panel url as an on page id. This prevents that.